### PR TITLE
agent statefulset

### PR DESCRIPTION
**Pull request by ramaraosrikakulapu**
**Requirement**: EC gateway HA in EKS

Changes in agent package for gateway HA loadbalancer

### CODE REVIEW

**Comment by ramaraosrikakulapu** [comment]
@ayasuda-ge can you please review the changes?

**Comment by ayasuda-ge** [comment]
> @ayasuda-ge can you please review the changes?

Please provide test/ci logs.

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # author: apolo.yasuda@ge.com
 
 language: minimal
-if: head_branch =~ ^v1.1beta_helm_pkg_update$ OR branch = v1.1beta_helm_pkg
+if: head_branch =~ ^v1.1beta_helm_pkg_update$ OR branch = v1.1beta_helm_pkg_update_multi_contnr_agent_lber
```
Comment (ayasuda-ge) [inline .travis.yml]: this shouldn't be modified. @

**Review by ayasuda-ge — COMMENTED** [review]
could you pinpoint is the test script for the lb? @ramaraosrikakulapu

**Comment by ramaraosrikakulapu** [comment]
> could you pinpoint is the test script for the lb? @ramaraosrikakulapu

@ayasuda-ge .. Current changes only for agent package. Tests for lb will be updated from `v1.1beta_helm_pkg_update_multi_contnr_lber` branch

File: .travis.yml (hunk 1 above)
Comment (ramaraosrikakulapu) [reply .travis.yml]: Branch name updated. However, if I update the branch name to `v1.1beta_helm_pkg`, its not triggering the CI. However [here](https://travis-ci.com/github/EC-Release/oci/builds/212900954) are the logs for all changes.

**Comment by ramaraosrikakulapu** [comment]
Closing this PR and new PR going to create from forked branch.

**Pull request closed without merging**